build/gulpfile.vscode.darwin.js

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+const gulp = require('gulp');
+const { signAsync } = require('electron-osx-sign');
+const util = require('./lib/util');
+const path = require('path');
+
+gulp.task('vscode-codesign', async () => {
```

Comment:
At this point, there's no reason for this to actually be a gulp task, we're not using any task dependencies nor streaming functionalities. Let's simply move all of this into a `build/darwin/sign.ts` file (typescript would be great) and we'd simply invoke it with `node build/darwin/sign.js`.